packages/bpk-component-image/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "private": true,
+  "name": "bpk-component-image",
+  "version": "0.0.0",
```

Comment:
So private is, we'll change that.
The unreleased number doesn't matter I think, as lerna will get us to assign a value when we do release it